packages/lib/bookingSuccessRedirect.ts

```diff
@@ -23,8 +23,11 @@ function getNewSeachParams(args: {
 
 type SuccessRedirectBookingType = Pick<
   BookingResponse | PaymentPageProps["booking"],
-  "uid" | "title" | "description" | "startTime" | "endTime" | "location" | "responses" | "references"
->;
+  "uid" | "title" | "description" | "startTime" | "endTime" | "location"
+> & {
+  references?: any;
+  responses?: any;
```

Comment:
No this is not correct. Don't set any as type, pls find the actual issue

Reply:
Currently the code is written in such way that this has to be done. If you know any other way to fix it go ahead.